pkg/metrics/metrics.go

```diff
@@ -114,6 +114,13 @@ var HcoMetrics = func() hcoMetrics {
 	}
 }()
 
+var hcoRecordingRules = []MetricDescription{
+	{`kubevirt_hyperconverged_operator_health_status`,
+		"Indicates whether HCO and its secondary resources health status is healthy (0), warning (1) or critical (2), based both on the firing alerts and resources conditions",
+		"Gauge",
```

Comment:
>"Indicates whether HCO and its secondary resources health status is healthy (0), warning (1) or critical (2), based both on the firing alerts and resources conditions"

"Indicates whether HCO and its secondary resources health status is healthy (0), warning (1) or critical (2), based both on the firing alerts, **that impact the operator health, and the kubevirt_hco_system_health_status metric**"